manifests/charts/gateway/templates/deployment.yaml

```diff
@@ -16,9 +16,10 @@ spec:
       {{- include "gateway.selectorLabels" . | nindent 6 }}
   template:
     metadata:
-      {{- with .Values.podAnnotations }}
       annotations:
-        {{- toYaml . | nindent 8 }}
+        istio.io/rev: {{ .Values.revision | default "default" }}
```

Comment:
This is already injected, we should not set it here. It actually would not be quite correct since `Values.revision` is the _requested_ revision here.

Reply:
Right, thanks. I forgot that this proxy will be injected.

Reply:
Done